src/utils/handlers/authenticationHandlers.js

```diff
@@ -89,10 +89,14 @@ export const getUser = () => {
   }
 };
 
-export const isLoggedIn = () => {
-  // This needs to be improved checking the token with decryption, checking payload for expiration
+export const isLoggedIn = (token) => {
+  // TODO: This needs to be improved checking the token with decryption, checking payload for expiration
```

Comment:
I would call getToken() inside of this function instead of having to pass it in every route as parameter. I can't think of a situation of passing the token as a parameter would be of help, and could also make it easy to forget to pass it, the token is the same everywhere. It would make sense to mock the token in tests, but it's not the case because they mock localStorage so the tests can find it as they would normally do.

Reply:
My reasoning for this approach was to keep the function pure (without any side-effects). I do see your point though. I'll make the changes. 👍